tutorials/erb_template.md

```diff
@@ -0,0 +1,129 @@
+\page erbtemplate ERB Template
+
+This tutorial shows how to place a large number of entites in a simulation world using ERB. ERB is the commonly used templating language to generate structured text files with Ruby code. In this case, since most simulation world used in gazebo is defined with sdformat (XML documents), ERB works conveniently.
```

Comment:
Consider adding a link to official ERB docs, i.e.

https://docs.ruby-lang.org/en/2.3.0/ERB.html